plugin/trino-mysql/src/test/java/io/trino/plugin/mysql/TestingMySqlServer.java

```diff
@@ -95,6 +102,14 @@ public void execute(String sql, String user, String password)
     @Override
     public void close()
     {
-        container.close();
+        try {
+            if (closeable != null) {
```

Comment:
Why nullable? 
it would be better if `startOrReuse` returned a close callback that is indempotent, instead of pushing this responsibility onto innocent callers like here

Reply:
actually i checked that and it seems the nuallbility is not needed here